package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "digitalmarketplace-frontend-toolkit",
   "description": "A toolkit for design patterns used across Digital Marketplace projects",
-  "version": "30.0.0",
+  "version": "30.0.1",
```

Comment:
I'd say this warrants a minor bump rather than a bugfix. It's a new feature we want the apps to use.